site/docs/concepts/taxonomy.md

```diff
@@ -0,0 +1,48 @@
+# Taxonomy
+
+Fybrik interacts with multiple external components, such as the data catalog, data governance policy manager, and modules.  In order for fybrik to orchestrate the data plane of a given workload it is essential that all the components involved use common terms.  For example, if a data governance policy refers to a particular transform it is crucial that the module implementing that transform refer to it in the same way, or for fybrik to be able to map between the disparate terms.
+
+A taxonomy defines the terms and related values that need to be commonly understood and supported across the components in the system:
+
+FybrikApplication yaml - information provided about the workload and the datasets
+Fybrik manager
+Data catalog
+Data Governance Policy Manager
+Config Policy Manager
+FybrikModules
+
```

Comment:
They appear in the same line in the "reading" version.

Reply:
fixed